requirements.txt

```diff
@@ -1,10 +1,10 @@
 black==22.3.0
-flake8==4.0.1
+flake8==5.0.4
 flake8-tidy-imports==4.6.0
-graviton@git+https://github.com/algorand/graviton@v0.3.0
+graviton@git+https://github.com/algorand/graviton@v0.5.0
 mypy==0.950
```

Comment:
Might be a good time to update `mypy` as well, but it would probably require an hour of two of little improvements.